gemini/__init__.py

```diff
@@ -1,8 +0,0 @@
-import os
```

Comment:
Thanks!  Aren't these imports required to properly import and use the GEMINI API in a Python script?